### DISCUSSION TITLE
Change existing window layout

### DISCUSSION THREAD

**Discussion by kuzyn007** [glzr-io/glazewm#175] [Q&A]
I understand that we can change layout (horizontal or vertical) for new opened apps. Can I move existing app to vertical/horizontal layout without changing all opened apps?
For existing window alt+v or shift+alt+v do not work. Also tried move shortcuts to change layout direction but all apps will change their direction together with one that im trying to move.

**Comment by ShoutsAtGoats** [comment]
I'd love an answer to this as well, I can't seem to get the window layout to change in vertical or horizonal. I understand it apparently has to do with new windows being added and possibly not existing ones, however this setting seems to do nothing.

**Comment by Purple-Tentacle** [comment]
There should be a symbol with 2 arrows in the bar. If not try this in your config
`- type: "tiling direction"
      label_horizontal: "⮂"
      label_vertical: "⮁"
      background: "#ffffff33"
      margin: "0 4px"
      padding: "0 8px"`

This symbol should change when you press ALT+v.
Then, new windows should be added below the current active one.
Windows already open can be moved and instead of replacing the window in the direction, it should be sorted below the one in the move direction.

To be more clear, if you want to want to move a window below another one, the tiling direction must be changed for the target window, not the window being moved.
